src/main/java/io/vertx/redis/RedisService.java

```diff
@@ -0,0 +1,1634 @@
+package io.vertx.redis;
+
+import io.vertx.codegen.annotations.ProxyGen;
+import io.vertx.codegen.annotations.ProxyIgnore;
+import io.vertx.codegen.annotations.VertxGen;
+import io.vertx.core.AsyncResult;
+import io.vertx.core.Handler;
+import io.vertx.core.Vertx;
+import io.vertx.core.json.JsonArray;
+import io.vertx.core.json.JsonObject;
+import io.vertx.redis.impl.RedisServiceImpl;
+import io.vertx.serviceproxy.ProxyHelper;
+
+@VertxGen
+@ProxyGen
+public interface RedisService {
+
```

Comment:
So my issue with this approach (wrapping all args in a JsonArray) is the service API isn't really buying me much as apposed to sending it manually over the event bus. And with redis being so simple (for most commands) get and set for example should be pretty simple.